docs/sections/user_guide/cli/tools/config.rst

```diff
@@ -284,229 +278,224 @@ and an additional supplemental YAML file ``values2.yaml`` with the following con
        date: 20240105
        empty: null
        greeting: Good Night
-       message: 'Good Night Moon Good Night Moon '
+       message: Good Night Moon Good Night Moon
```

Comment:
How did we get rid of all these quotes around the messages? I wasn't sure where this change was coming from, or if this was from an older PR and just needed to be updated.

Reply:
That was done [here](https://github.com/ufs-community/uwtools/pull/493/files#diff-a4378ed914a7ad3423e757c932d624dff449acbaff2574aaed9ec48e57c29c93R207).